kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue

```diff
@@ -1043,7 +1043,8 @@
         context: 'Label for enabled pages section',
       },
       enabledPagesDescription: {
-        message: 'Unselect a page to hide it even if the user has permission to access it.',
+        message:
+          'When you uncheck a page, it will make it invisible to users even if they have permission to access it.',
```

Comment:
Change to:
`When you uncheck a page, it will become invisible to users even if they have permission to access it.`